composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
         "zendframework/zend-loader": "~2.2.5",
         "zendframework/zend-modulemanager": "~2.2.5",
         "zendframework/zend-servicemanager": "~2.2.5"
```

Comment:
Hey @DASPRiD , 

It looks like zendframework/zend-servicemanager is the dependency and not the `zendframework/zend-stdlib` . I did checked the code and there was no reference to StdLib .